x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/ilm/action/TransportExplainLifecycleActionTests.java

```diff
@@ -85,7 +85,8 @@ public void testGetIndexLifecycleExplainResponse() throws IOException {
                 true,
                 true,
                 indexLifecycleService,
-                REGISTRY
+                REGISTRY,
+                false
```

Comment:
Should we also add a test where the last parameter is `true` ?

Reply:
I intentionally didn't add an extra test, as the underlying function that actually uses that parameter (the static `applyDefaultConditions`) is already properly tested. But now that you I'm thinking about it, I think it might make sense to make these all `randomBoolean()`. Would you agree?

Reply:
Sure. That makes sense. 

IMO, since this test is about the explain lifecycle response, it'd be nice to have one test that validates that the `true` value of the boolean does something to the response (i.e. adds some conditions) - this test would ensure that `TransportExplainLifecycleAction` _actually calls_ `applyDefaultConditions` when that flag is `true`